### DISCUSSION TITLE
How to add side pane navigation table of contents?

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#3291] [Q&A]
Consider the code below. It produces the table of contents correctly. However, it does not create a table of contents that one can navigate by using the side pane of the `pdf` viewer. (Please, see what I mean in the image below.) Is that possible with `typst`? Thanks!

```
#set heading(numbering: "1.")

#show outline.entry.where(
  level: 1
): it => {
  v(12pt, weak: true)
  strong(it)
}

#outline(indent: auto)

= Introduction
= Background
== History
== State of the Art
= Analysis
== Setup
```

![image](https://github.com/typst/typst/assets/75945439/ccd10e58-a4c9-4973-a202-a84f82698446)

**Comment by laurmaedje** [comment]
That should happen automatically and also does so for me. Which PDF viewer are you using and could you try whether it shows up in a different one?

**Reply by psads-git** [reply]
I am using

`Evince`.

However, in `Okular`, the navigation side pane appears.

Strange!

**Reply by psads-git** [reply]
My fault, @laurmaedje: I forgot to click on the right icon in `Evince` (that is a very small icon).

Soon, I will delete this question.

**Reply by Atreyu-94** [reply]
Gnome minimalism is too rough lately xD
